modules/microshift-disconnected-host-procedure.adoc

```diff
@@ -0,0 +1,128 @@
+----
+
+. Get the hostname of the device by running the following command:
++
+[source,terminal]
+----
+$ NAME="$(hostnamectl hostname)"
+----
+
+. Add an entry for the hostname of the node in the `/etc/hosts` file by running the following command:
++
+[source,terminal]
+----
+$ sudo echo "$IP $NAME" >> /etc/hosts
+----
+
+. Update the {microshift-short} configuration file by adding the following YAML snippet to `/etc/microshift/config.yaml`:
++
+[source,terminal]
+----
+sudo cat <<EOF >> /etc/microshift/config.yaml
+node:
+  hostnameOverride: “$NAME”
+  nodeIP: “$IP”
+EOF
+----
+
+. If the device currently has a NIC attached, it can be removed. Determine the interface with the default route by running the following command:
++
+[source,terminal]
+----
+$ IFACE=$(ip route | grep default | awk '{print $5}')
+----
+
+. Stop external network access by running the following commands:
++
+[source,terminal]
+----
+$ sudo nmcli connection modify $IFACE connection.autoconnect no
+----
++
+[source,terminal]
+----
+$ sudo nmcli connection down $IFACE
+----
+
+. Restart the {microshift-short} host to apply the configuration changes by running the following command:
++
+[source,terminal]
+----
+$ sudo systemctl reboot <1>
+----
+<1> This step restarts the cluster. Wait for the greenboot health check to report the system healthy before implementing verification.
+
+.Verification
+
+At this point, network access to the {microshift-short} host has been severed. Use the host CLI to verify that the cluster has started in a stable state.
+
+. Verify that the {microshift-short} cluster is running by entering the following command:
++
+[source,terminal]
+----
+$ export KUBECONFIG=/var/lib/microshift/resources/kubeadmin/kubeconfig
+$ sudo -E oc get pods -A
+----
++
+.Example output
+[source,terminal]
+----
+NAMESPACE                  NAME                                       READY   STATUS    RESTARTS      AGE
+kube-system                csi-snapshot-controller-74d566564f-66n2f   1/1     Running   0             24h
+kube-system                csi-snapshot-webhook-69bdff8879-xs6mb      1/1     Running   0             24h
+openshift-dns              dns-default-dxglm                          2/2     Running   0             24h
+openshift-dns              node-resolver-dbf5v                        1/1     Running   0             24h
+openshift-ingress          router-default-8575d888d8-xmq9p            1/1     Running   0             24h
+openshift-ovn-kubernetes   ovnkube-master-gcsx8                       4/4     Running   1 (24h ago)   24h
+openshift-ovn-kubernetes   ovnkube-node-757mf                         1/1     Running   1 (24h ago)   24h
+openshift-service-ca       service-ca-7d7c579f54-68jt4                1/1     Running   0             24h
+openshift-storage          topolvm-controller-6d777f795b-bx22r        5/5     Running   0             24h
+openshift-storage          topolvm-node-fcf8l                         4/4     Running   0             24h
```

Comment:
Woops. Output show reflect that cluster just started, rather than being up for 24hrs prior 😅  
```suggestion
NAMESPACE                  NAME                                       READY   STATUS    RESTARTS      AGE
kube-system                csi-snapshot-controller-74d566564f-66n2f   1/1     Running   0             1m
kube-system                csi-snapshot-webhook-69bdff8879-xs6mb      1/1     Running   0             1m
openshift-dns              dns-default-dxglm                          2/2     Running   0             1m
openshift-dns              node-resolver-dbf5v                        1/1     Running   0             1m
openshift-ingress          router-default-8575d888d8-xmq9p            1/1     Running   0             1m
openshift-ovn-kubernetes   ovnkube-master-gcsx8                       4/4     Running   1             1m
openshift-ovn-kubernetes   ovnkube-node-757mf                         1/1     Running   1             1m
openshift-service-ca       service-ca-7d7c579f54-68jt4                1/1     Running   0             1m
openshift-storage          topolvm-controller-6d777f795b-bx22r        5/5     Running   0             1m
openshift-storage          topolvm-node-fcf8l                         4/4     Running   0             1m
```